app/cdap/components/SnackbarComponent/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2017-2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import * as React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import { useErrorStyles } from './styles';
+import { State } from './types';
+import { TransitionComponent } from './Components/TransitionComponent';
+
+const PositionedSnackbar = (props) => {
+  const classes = useErrorStyles();
+  const [state, setState] = React.useState<State>({
```

Comment:
IState or name this ISnackBarState orsomethign - also import {useState}

Reply:
why are thse types combined into one state? why isn't isOpen its own piece? do the horizontal and vertical values change?

Reply:
Done